src/detectLaunchConfigurationChanges.ts

```diff
@@ -17,6 +17,7 @@ export function detectLaunchConfigurationChanges(
       "serverProperties",
       "javaHome",
       "customRepositories",
+      "coursierMirror",
```

Comment:
Can we extract these keys as constants and use them in vscode?

Reply:
In a couple of places yes, but unfortunately not in `package.json` which would be most useful

Reply:
That's true, but at least we can reduce 3 possible sources of errors (config options names) to just 2